libs/langchain/langchain/retrievers/kendra.py

```diff
@@ -336,13 +354,18 @@ class AmazonKendraRetriever(BaseRetriever):
     page_content_formatter: Callable[[ResultItem], str] = combined_text
     client: Any
     user_context: Optional[Dict] = None
+    score_confidence: float = 0
```

Comment:
Could we rename to `min_score_confidence` and use an Optional[Float] = None, i'd prefer making it obvious that no filter is being applied via `None` rather than via `0`